src/widgets/component-new-edit/components/response-format/simple/response-format-simple.jsx

```diff
@@ -58,11 +60,16 @@ class ResponseFormatSimple extends Component {
           <View key={TEXT} value={TEXT} label={Dictionary.TEXT}>
             <ResponseFormatDatatypeText />
           </View>
-          <View key={DATE} value={DATE} label={Dictionary.DATE} />
+          <View key={DATE} value={DATE} label={Dictionary.DATE} >
+          <ResponseFormatDatatypeDate />
```

Comment:
Do you format your code ? you should use the built-in prettier formatting tool we have on this project.

Reply:
@EmmanuelDemey  i didnt format my code and i used simple-date.jsx file to insert the new input